zoo/entities/migrations/0001_initial.py

```diff
@@ -0,0 +1,150 @@
+# Generated by Django 2.2.19 on 2021-07-01 11:30
```

Comment:
once more, migration name 😄 
and I think we should only have a migration on this commit, as we're just adding the models here, so let's squash them into one